multiples_sum.js

```diff
@@ -0,0 +1,23 @@
+function multSum(valOne, valTwo, max) {
+  var sumOne = 0;
+  var sumTwo = 0;
+  var numOne = 1;
+  var numTwo = 1;
+  // finding all multiples of valOne
+  do {
+    var prodOne = valOne * numOne;
+    numOne++;
+    sumOne += prodOne;
+  } while (prodOne < max);
+  // finding all multiples of valTwo
+  do {
+    var prodTwo = valTwo * numTwo;
+    numTwo++;
+    sumTwo += prodTwo;
+  } while (prodTwo < max);
+  var sum = sumOne + sumTwo;
```

Comment:
I suspect this algorithm can be condensed to a single easier-to-read loop using less variables if we use the `remainder` aka `modulo` operator. [Docs on this operator here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Remainder). Consider something like:

````
// Find the sum of all numbers below 1000 that are a multiple of 3 and/or 5

let sum = 0;
function calcMultiplesSum() {
  for(let i = 3; i < 1000; i++) {
    if(i % 3 === 0 || i % 5 === 0) {
      sum += i;
    }
  }

  return sum;
}

console.log(calcMultiplesSum());
````

I haven't tested this locally and do not have this problem saved so you'll have to test this yourself.

Reply:
This is great. I'd sort of given up on finding the most efficient route so I sort of slapped together my long form code. And you're right, I see now that my sum variable was capturing duplicates of shared multiples. Thanks Tanner!